pkg/controller/performanceprofile/components/machineconfig/machineconfig.go

```diff
@@ -188,6 +242,30 @@ func getSystemdContent(options []*unit.UnitOption) (string, error) {
 	return string(outBytes), nil
 }
 
+func getCrioHooksConfigContent(configFile string, profile *performancev2.PerformanceProfile) ([]byte, error) {
```

Comment:
nit: use upper case letters for acronyms - getCRIOHooksConfigContent

Reply:
Missed that one, thanks